lightstep/resource_metric_condition.go

```diff
@@ -488,9 +516,7 @@ func (p *resourceUnifiedConditionImp) resourceUnifiedConditionImport(ctx context
 }
 
 func getUnifiedConditionAttributesFromResource(d *schema.ResourceData, schemaType ConditionSchemaType) (*client.UnifiedConditionAttributes, error) {
-	expression := d.Get("expression").([]interface{})[0].(map[string]interface{})
-
-	thresholds, err := buildThresholds(d)
```

Comment:
`buildThresholds` makes assumptions about `expression` being present and ends up in the `Expression` so I moved it inside `buildExpression`